src/storage/txn/actions/flashback_to_version.rs

```diff
@@ -96,7 +96,14 @@ pub fn flashback_to_version<S: Snapshot>(
             *next_lock_key = Some(key);
             break;
         }
-        txn.unlock_key(key.clone(), lock.is_pessimistic_txn());
+        rollback_lock(
+            txn,
+            reader,
```

Comment:
We should set `reader.start_ts` to `lock.ts` each time before calling `rollback_lock` as discussed previously.

Reply:
okay. I will do that.